packages/jest-cli/src/reporters/DefaultTestReporter.js

```diff
@@ -139,6 +140,37 @@ class DefaultTestReporter {
         `${PENDING_COLOR(`${print('test', pendingTests)} skipped`)}, `;
     }
 
+    let snapshotsAdded = 0;
+    let snapshotsFiles = 0;
+    let snapshotsMatched = 0;
+    let snapshotsRemoved = 0;
+    aggregatedResults.testResults.forEach(result => {
+      if (result.snapshotsAdded) {
+        snapshotsFiles++;
+      }
+      snapshotsAdded += result.snapshotsAdded;
+      snapshotsMatched += result.snapshotsMatched;
+      snapshotsRemoved += result.snapshotsRemoved;
+    });
+    if (snapshotsAdded || snapshotsMatched) {
+      results += `${SNAPSHOT_SUMMARY('Snapshot Summary')}\n`;
+      if (snapshotsAdded) {
+        results +=
+          `* ${snapshotsAdded} ${pluralizeSnapshot(snapshotsAdded)} written ` +
+          `in ${snapshotsFiles} test file${snapshotsFiles > 1 ? 's' :''}\n`;
+      }
+      if (snapshotsRemoved) {
+        results +=
+          `* ${snapshotsRemoved} ${pluralizeSnapshot(snapshotsRemoved)} `+
+          `deleted\n`;
+      }
+      if (snapshotsMatched) {
+        results +=
```

Comment:
should we make this green?

I don't know if this is gonna look clowny with so many colors or not. Up to you to experiment making the whole sentence yellow or just the `5 snapshots` part or nothing at all :)